src/jdk.jdi/windows/native/libdt_shmem/shmem_md.c

```diff
@@ -37,7 +37,7 @@
  * yet.
  */
 
-static HANDLE memHandle = NULL;
+// static HANDLE memHandle = NULL;
```

Comment:
I still think this should be removed instead of commented out.

Reply:
I was more fond of keeping it as a comment, but oh well. Give me a moment...